client/v3/client.go

```diff
@@ -52,6 +52,7 @@ type Client struct {
 	Watcher
 	Auth
 	Maintenance
+	Kubernetes Kubernetes
```

Comment:
etcd sdk is used by many users, including Kubernetes.  Some users may not care about the `Kubernetes` interface at all. It doesn't make much sense to expose `Kubernetes` interface to users other than Kubernetes or Controllers.

How about this? 
```
package kubernetes

func New(c *Client) Kubernetes {
	return &kubernetesClient{
		Client: c,
		kv: RetryKVClient(c)
	}
}

type kubernetesClient struct {
	Client
	kv pb.KVClient
}

......
```